### ISSUE TITLE
(BC19) ALOpsAppCompiler@2 - Provide support for resourceExposurePolicy (instead of ShowMyCode)

### ISSUE THREAD

**Issue by fvet** [HodorNV/ALOps#431]
With the deprecation of ShowMyCode (app.json) in BC19, we'd like to see support to set the **resourceExposurePolicy** properties via ALOpsAppCompiler@2. This would allow us - same way as we used ShowMyCode now - to have different behavior / output when targetting AppSource vs OnPrem apps.

"resourceExposurePolicy": {
    "allowDebugging": true
    "allowDownloadingSource": true
    "includeSourceInSymbolFile": true
  }

**Comment by waldo1001** [comment]
It's indeed on the agenda.

**Comment by fvet** [comment]
@waldo1001 Any ETA when to expect the next ALOps update (including this issue)?

**Comment by waldo1001** [comment]
We're going to try  to get this in by next week.  It's not that difficult, but backwards compatibility might make things a bit harder.. .

**Comment by fvet** [comment]
Fixed in v1.446

```
resourceexposurepolicy_allowdebugging: 'Enable'
resourceexposurepolicy_allowdownloadingsource: 'Disable'
resourceexposurepolicy_includesourceinsymbolfile: 'Disable'
```
